Makefile

```diff
@@ -1,11 +1,38 @@
 PACKAGES_NOSIMULATION=$(shell go list ./... | grep -v '/simulation')
 BINDIR ?= $(GOPATH)/bin
+VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
+COMMIT := $(shell git log -1 --format='%H')
 
 export GO111MODULE = on
 
 include Makefile.ledger
 
-export GO111MODULE = on
```

Comment:
I suggest keeping this in case a user has installed the project inside GOPATH

Reply:
Uhm, I only removed one of the two instances of `GO111MODULE`, having it before _and_ after `include Makefile.ledger` shouldn't make any difference.

Reply:
Sorry, didn't notice it was doubled